src/routes/settings-knowledge-base.js

```diff
@@ -164,8 +169,14 @@ class SettingsKnowledgeBaseRoute extends Component {
     const {
       kbCredentials,
       intl,
+      stripes,
+      history,
     } = this.props;
 
+    if (!stripes.hasPerm('ui-eholdings.settings.kb')) {
+      history.push('/settings/eholdings/knowledge-base');
```

Comment:
If want to hide the third pane with Kb settings you should redirect to `/settings/eholdings`

Reply:
Fixed